float64 time_stamp
uint8 log_type
string log_str
